pom.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>23</version>
+    </parent>
+
+    <artifactId>amoro-shade</artifactId>
+    <modules>
+        <module>amoro-shade-guava-32</module>
+        <module>amoro-shade-jackson-parent</module>
+    </modules>
+    <groupId>org.apache.amoro</groupId>
+    <name>amoro-shade</name>
+    <version>0.7-SNAPSHOT</version>
```

Comment:
I checked the flink and paimon projects, and the shade project has a separate version. Defined as `paimon.shade.version` and `flink.shaded.version` in the project respectively. So I don’t think it needs to be consistent with the amoro project.

Reply:
@zhoujinsong @klion26

Reply:
IMHO the version in the shaded project doesn't need to respect the version of the main repo, because their update frequency is completely different. But I'm fine with the either way.
Seems flink-shade doesn't align the version with the main repo, paimon aligns this version with the main repo.

Reply:
I saw that the latest version paimon is also unaligned, so aligned version is meaningless. 
And keep its self release version will be fine.

Reply:
Yes, the version of paimom-shade does not respect the main repo for now. 
For `paimon aligns this version with the main repo` I mean that when paimon-shade bumps its version, it will repesct the *current* version of paimon.